Mathlib/RepresentationTheory/GroupCohomology/Basic.lean

```diff
@@ -87,113 +87,150 @@ abbrev linearYonedaObjResolution (A : Rep k G) : CochainComplex (ModuleCat.{u} k
     ((((linearYoneda k (Rep k G)).obj A).rightOp.mapHomologicalComplex _).obj (resolution k G))
 #align group_cohomology.linear_yoneda_obj_resolution GroupCohomology.linearYonedaObjResolution
 
-theorem linearYonedaObjResolution_d_apply {A : Rep k G} (i j : ℕ) (x : (resolution k G).pt i ⟶ A) :
+theorem linearYonedaObjResolution_d_apply {A : Rep k G} (i j : ℕ) (x : (resolution k G).X i ⟶ A) :
     (linearYonedaObjResolution A).d i j x = (resolution k G).d j i ≫ x :=
   rfl
 #align group_cohomology.linear_yoneda_obj_resolution_d_apply GroupCohomology.linearYonedaObjResolution_d_apply
 
-end groupCohomology
+end GroupCohomology
 
 namespace InhomogeneousCochains
 
-open Rep groupCohomology
+open Rep GroupCohomology
 
 /-- The differential in the complex of inhomogeneous cochains used to
 calculate group cohomology. -/
 @[simps]
 def d [Monoid G] (n : ℕ) (A : Rep k G) : ((Fin n → G) → A) →ₗ[k] (Fin (n + 1) → G) → A where
   toFun f g :=
     A.ρ (g 0) (f fun i => g i.succ) +
-      Finset.univ.Sum fun j : Fin (n + 1) =>
+      Finset.univ.sum fun j : Fin (n + 1) =>
         (-1 : k) ^ ((j : ℕ) + 1) • f (Fin.contractNth j (· * ·) g)
   map_add' f g := by
     ext x
-    simp only [Pi.add_apply, map_add, smul_add, Finset.sum_add_distrib, add_add_add_comm]
+/- Porting note: changed from `simp only` which needed extra heartbeats -/
+    simp_rw [Pi.add_apply, map_add, smul_add, Finset.sum_add_distrib, add_add_add_comm]
   map_smul' r f := by
     ext x
-    simp only [Pi.smul_apply, RingHom.id_apply, map_smul, smul_add, Finset.smul_sum, ← smul_assoc,
+/- Porting note: changed from `simp only` which needed extra heartbeats -/
+    simp_rw [Pi.smul_apply, RingHom.id_apply, map_smul, smul_add, Finset.smul_sum, ← smul_assoc,
       smul_eq_mul, mul_comm r]
 #align inhomogeneous_cochains.d InhomogeneousCochains.d
 
 variable [Group G] (n) (A : Rep k G)
 
+/- Porting note: linter says the statement doesn't typecheck -/
```

Comment:
CI gives a green checkmark, so I suppose that the linter is no longer complaining?

Reply:
That's because I added a `nolint checkType`. I'll mention that in the porting note.